packages/web-app-files/src/index.js

```diff
@@ -72,6 +73,13 @@ const navItems = [
       path: `/${appInfo.id}/shares/via-link`
     }
   },
+  {
```

Comment:
Can't be a static nav item. You need to check the capabilities if the spaces feature is enabled. At the moment this nav item will be visible in an oc10 deployment as well.

Reply:
There is already a `enabled` prop available, so we used this one (see https://github.com/owncloud/web/pull/6199/files#diff-596b5f5cdc5da8df260bef258a7439ac877ae23495400c227bb167954fdabcb1R82). Technically the route is still accessible, but not showing in the side bar. Is this sufficient, or should be prevent access to it as well?